requirements/dev.txt

```diff
@@ -33,3 +32,9 @@ transifex-client==0.12.4
 
 # Dependency of transifex-client
 urllib3==1.20
+
+# Install the Zulip API bindings into the virtualenv; note that this
+# won't be automatically updated for new versions of api/.  This is
+# used in development for some tests and in production for things like
+# Nagios checks.
+#api/
```

Comment:
(1) another constraint that `pip-compile` doesn't know how to fetch from local dir
(2) I somehow missed that